client/ayon_core/hosts/houdini/plugins/publish/collect_files_for_cleaning_up.py

```diff
@@ -0,0 +1,112 @@
+import pyblish.api
+import os
+from ayon_core.pipeline import AYONPyblishPluginMixin
+
+
+class CollectFilesForCleaningUp(pyblish.api.InstancePlugin,
+                                AYONPyblishPluginMixin):
+    """Collect Files For Cleaning Up.
+    
+    This collector collects output files
+    and adds them to file remove list.
+
+    CAUTION:
+        This collector deletes the exported files and
+          deletes the parent folder if it was empty.
+        Artists are free to change the file path in the ROP node.
+    """
+
+    order = pyblish.api.CollectorOrder + 0.2  # it should run after CollectFrames
+
+    hosts = ["houdini"]
+    families = [
+        "camera",
+        "ass",
+        "pointcache",
+        "imagesequence",
+        "mantraifd",
+        "redshiftproxy",
+        "review",
+        "staticMesh",
+        "usd",
+        "vdbcache"
+    ]
```

Comment:
Do we want to expose these in settings so studio can decide per family whether they want to trigger this or not?

Reply:
That's fair!